6b-precompiled-http-to-service-bus-topic/.vscode/launch.json

```diff
@@ -0,0 +1,11 @@
+{
+    "version": "0.2.0",
+    "configurations": [
+        {
+            "name": "Attach to .NET Functions",
+            "type": "coreclr",
+            "request": "attach",
+            "processId": "${command:azureFunctions.pickProcess}"
+        }
+    ]
+}
```

Comment:
Could you please remove the `.vscode` folder and add it to `.gitignore`?

Reply:
Sure, I updated the PR accordingly.